todo_app/tests/test_app.py

```diff
@@ -0,0 +1,94 @@
+from dotenv import load_dotenv, find_dotenv
```

Comment:
Minor, but I'd suggest adding an empty `__init__.py` file to this folder. It often proves unimportant (you're probably not seeing any issues without it currently, and I had no issues running this) but there are scenarios where it being present is necessary for Python to recognise the folder as a module and allow local imports to work correctly; in particular it _might_ come back to bite you when we try to run our tests in Docker later on with some slightly cryptic error messages